lib/importer/factory/object_factory.rb

```diff
@@ -133,6 +133,15 @@ def remote_files
         end
       end
 
+      def sanitized_attributes
+        attributes
+          .slice(*permitted_attributes)
+          .each_with_object({}) do |(k, v), memo|
+          value = Array(v)          
+          memo[k] = klass.multiple?(k) ? value : value.first
```

Comment:
I think this needs something more nuanced than `klass.multiple?(k)` since not all the keys in the `attributes` are ones for which this method works.  See the Travis failures.